src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-snapshot-list/rbd-snapshot-list.component.spec.ts

```diff
@@ -206,7 +206,7 @@ describe('RbdSnapshotListComponent', () => {
     it('should display suggested snapshot name', () => {
       component.openCreateSnapshotModal();
       expect(component.modalRef.content.snapName).toMatch(
-        RegExp(`^${component.rbdName}-\\d+T\\d+Z\$`)
+        RegExp(`^${component.rbdName}_[0-9-]+T[0-9.:]+\\+[0-9:]+\$`)
```

Comment:
I favor `\\d`

Reply:
Let's stick to using [ISO_8601](https://en.wikipedia.org/wiki/ISO_8601) date formats, please.

Reply:
That might be a missunderstanding. I just said that I prefer another way of writing the regular expression. I don't want to change the format. `\d` is just a group of characters, in that case, digits. It can equivalently be used for `[0-9]`, also see [matching characters](https://docs.python.org/2/howto/regex.html#matching-characters). As this PR changes that (`\d` has been used before) and prefers writing `[0-9]` over using that group, I thought I'd just drop my thought on that.

Reply:
Done

@LenzGr it sticks to `2019-04-23T11:50:41+00:00` in order to preserve the local time of the user that creates the snapshot.